lib/Routing/RedirectHandler.php

```diff
@@ -261,10 +261,6 @@ protected function buildRedirectResponse(Redirect $redirect, Request $request, $
      */
     private function getRegexRedirects()
     {
-        if (null !== $this->redirects && is_array($this->redirects)) {
```

Comment:
It leads to more hits to the cache rather than returning from cached property..maybe use default value `null`?

Reply:
The `|null` is missing at the attribute php doc.

Reply:
I added the null to the docs and added the check and the return again